private/buf/buffetch/format.go

```diff
@@ -14,6 +14,33 @@
 
 package buffetch
 
+// TODO: remove the unexported formats below and only use these ones.
+// A good time to do so is when we merge txtpb and binpb into this branch.
+const (
+	// formatBin is the binary format.
```

Comment:
Instead of renaming `formatXyz` to `FormatXYZ`, which will produce a lot of diff in this PR, I added `FormatXyz` as a temporary measure.
This will be cleaned up when we merge main into this branch, which will introduce `txtpb` and `binpb` to this branch.